business_requirement_deliverable/tests/test_br.py

```diff
@@ -0,0 +1,248 @@
+            price_list = line._get_pricelist()
+            self.assertEqual(
+                price_list.id,
+                self.partner.property_product_pricelist.id)
+
+    def test_product_id_change(self):
+        for line in self.br.deliverable_lines:
+            pr_mock0 = self.env['business.requirement.deliverable'].new()
+            pr_mock0.product_id = self.productA.id
+            description = ''
+            unit_price = 0
+            product = self.productA
+
+            if product:
+                description = product.name_get()[0][1]
+                unit_price = product.list_price
+
+            if product.description_sale:
+                description += '\n' + product.description_sale
+            unit_price = pr_mock0.product_id.list_price
+            pricelist = pr_mock0._get_pricelist()
+            if pricelist:
+                product = pr_mock0.product_id.with_context(
+                    lang=pr_mock0.business_requirement_id.partner_id.lang,
+                    partner=pr_mock0.business_requirement_id.partner_id.id,
+                    quantity=pr_mock0.qty,
+                    pricelist=pricelist.id,
+                    uom=pr_mock0.uom_id.id,
+                )
+                unit_price = product.price
+
+            if pricelist:
+                product = pr_mock0.product_id.with_context(
+                    lang=pr_mock0.business_requirement_id.partner_id.lang,
+                    partner=pr_mock0.business_requirement_id.partner_id.id,
+                    quantity=pr_mock0.qty,
+                    pricelist=pricelist.id,
+                    uom=pr_mock0.uom_id.id,
+                )
+                unit_price = product.price
+            pr_mock0.product_id_change()
+            self.assertEqual(pr_mock0.name, description)
+            self.assertEqual(pr_mock0.uom_id.id, self.productA.uom_id.id)
+            self.assertEqual(pr_mock0.unit_price, unit_price)
+
+    def test_product_uom_change(self):
+        self.uom_id = self.env['product.uom'].search([('id', '=', 3)])
+        for line in self.br.deliverable_lines:
+            pr_mock = self.env['business.requirement.deliverable'].new()
+            pr_mock.product_id = self.productA.id
+            pr_mock.product_id_change()
+            self.unit_price = pr_mock.unit_price
+            pr_mock1 = self.env['business.requirement.deliverable'].new()
+            pr_mock1.product_id = self.uom_id.id
+            pr_mock1.product_uom_change()
+            self.assertTrue(line.unit_price > self.unit_price)
+
+    def test_partner_id_change(self):
+        self.partner = self.env['res.partner'].create({
+            'name': 'Your company test',
+            'email': 'your.company@your-company.com',
+            'customer': True,
+            'company_type': 'company',
+        })
+        self.br.write({'partner_id': self.partner.id})
+        try:
+            self.br.partner_id_change()
+        except UserError, e:
```

Comment:
flake8 returns this error:
`business_requirement_deliverable/tests/test_br.py:247:26: E901 SyntaxError: invalid syntax`

Could you check?